src/content/guides/integrating-third-party-asset.mdx

```diff
@@ -108,6 +108,29 @@ curl https://auth.centrapay.com/api/.well-known/jwks.json
 }
 ```
 
+To verify a request start by decripting the JWT.
+
+**Decripted JWT Payload**
+
+```JSON
+{
+  "iat": "1684105185",
+  "exp": "1684105485",
+  "aud": "https://your.endpoint/",
+  "request_body_sha256": "b9195bf41bf0e38ab0ab44e7ef5b9af5cb0fe2ece8dee5d112d7485bf4ef0007",
+}
+```
+
+Use the decripted JWT validate the following fields:
+
+|        Field        |                                                                         Assertion                                                                           |
+| ------------------- | ----------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| exp                 | Assert that exp is greater than the current Unix timestamp.                                                                                                 |
```

Comment:
@laurencemcknight I thought we were including the `iat` field too?

We should document what our expiry is (e.g. 5 mins from the time that the JWT was created) + provide the `iat` field to allow integrators to use a shorter expiry if they wish